packages/framework/client-logger/fluid-telemetry/package.json

```diff
@@ -77,10 +77,14 @@
 		"format:prettier": "prettier --write . --cache --ignore-path ../../../../.prettierignore",
 		"lint": "fluid-build . --task lint",
 		"lint:fix": "fluid-build . --task eslint:fix --task format",
+		"start:tinylicious:test": "tinylicious > tinylicious.log 2>&1",
 		"test": "npm run test:mocha",
 		"test:mocha": "npm run test:mocha:esm && echo skipping cjs to avoid overhead - npm run test:mocha:cjs",
```

Comment:
As things are configured now, if a developer runs `npm run test` (or `test:mocha`, etc.) the tests will fail due to tinylicious not running, right? If so, we may want to reconfigure the scripts to ensure they are all runnable.

I also don't think we want `test:mocha` (etc.), as it will result in the tests being run as a part of the global `test:mocha` run, which won't be correctly set up for tinylicious to run.

I would recommend removing the mocha script variants, and update `test` to call `test:realsvc`

Reply:
That said, there might also be a case for having 2 separate suites of tests here:
1. The end-to-end tests that run with tinylicious via `test:realsvc`
2. The simple unit tests that run via `test:mocha`

Then we would just want to update `test` to call both of them for convenience.

Reply:
I also like the idea of two separate suites/kinds of tests. Keep the actual unit tests running under `test:mocha` so CI only runs those, and `test` can kick off both those and the realsvc ones. Although if we want the realsvc ones to run in CI, we'd need a `test:realsvc:tinylicious:report` script... which currently nothing has except the e2e tests package, so I'm hesitant to add it here. I'd be ok just triggering them locally through `test` for now.

How to split the two kinds of tests... I might go with `*.spec.ts` for the unit ones, and maybe `.spec.realsvc.ts` for the other ones, and pass the appropriate filter to mocha in each of the scripts? Although that's not something we do in other places as far as I know, so maybe split them into two subfolders of src/test/ with `*.spec.ts` suffix in both cases?

Reply:
Updated with split tests.